specificatie/openapi.yaml

```diff
@@ -483,6 +603,66 @@ paths:
         default:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default'
 
+  #333, 335 
+  /panden/_zoek:
+    post:
+      requestBody:
+        description: Geometrie waarbinnen naar panden gezocht moet worden.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PandenZoekObject'
+      security:
+        - apiKeyBAG: []
+      operationId: zoekPandenInContour
+      tags:
+        - Pand
+      summary: "Vindt panden die binnen de contour vallen en die voldoen aan de opgegeven status en geconstateerd."
+      description: "Zoek binnen een geometrische contour naar panden die voldoen aan de opgegeven status en geconstateerd. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.1.0/features/fields.feature)."
```

Comment:
Ik mis hier de mogelijkheid om te zoeken op bouwjaar.

Reply:
In de meest recente commit is het bouwjaar wel opgenomen in het PandenZoekObject met een min en een max....

Reply:
ik zie het wel bij de parameters, maar niet in de description. Maar ik heb enige moeite het totale overzicht en de laatste versie te krijgen, dus het kan aan mij liggen.

Reply:
Aangepast.